src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = sourceString.split(';');
+  const cssCode = {};
```

Comment:
The variable name 'cssCode' is not very descriptive of its purpose as a storage for the parsed CSS properties. A more descriptive name like 'styleObject' or 'cssObject' would improve readability.